### DISCUSSION TITLE
rounded-s-lg or rounded-e-lg not working

### DISCUSSION THREAD

**Discussion by KhairulAzmi21** [andrewdwallo/filament-selectify#17] [Q&A]
![Screenshot 2024-03-04 at 09 27 26](https://github.com/andrewdwallo/filament-selectify/assets/12378983/41e796ee-6e4f-4a46-9b69-d9810de0c820)

Hi @andrewdwallo , do you have any ideas why border radius was not working ?

**Comment by andrewdwallo** [comment]
Did you make sure to create a Filament theme and import my packages vendor folder in tailwindcss config file?
